docs/content/documentation/operations/release-management.md

```diff
@@ -269,8 +269,12 @@ the `gh-pages` branch.
 > `main` branch. Older versions should be snapshotted in the `gh-pages` branch
 > using the workflow described above.
 
+### 11. Update Helm Documentation
 
-### 11. Add a Snapshot of the Current Documentation
+If you have updated inline documentation in helm charts, make sure to reflect
+the changes by running `./hack/helm-docs.sh`.
```

Comment:
This function just calls `helm-docs` with a tiny wrapper to warn the user to install `helm-docs` if the binary cannot be found.